docs/admin/backup.rst

```diff
@@ -0,0 +1,177 @@
+Veracrypt).
+
+Navigate to **Q ▸ System Tools ▸ Backup Qubes**, and move all VMs from 
+"Selected" to "Available" by pressing the ``<<`` button. 
+
+To target a VM for backup, highlight it and move it into the "Selected" 
+column by pressing the ``>`` button. Select the following VMs:
+
+ - ``dom0``
+ - ``vault``
+ - any customized VMs that you wish to preserve.
+
+You do not need to back up the ``sd-`` VMs.
+
+Click "Next", and in "Backup destination," specify the VM and directory 
+corresponding to your storage medium's current mount point.
+
+Set a strong, unique backup passphrase (7-word diceware), and ensure this 
+passphrase is stored securely outside SecureDrop Workstation. 
+
+.. note:: 
+ This passphrase protects sensitive 
+ components of your SecureDrop instance, including the *Submission Private Key*.
+ Ensure that it is a very strong password and is stored securely.   
+
+Uncheck "save backup profile," then proceed with the backup.
+
+QubesOS recommends verifying the integrity of the backup once the backup 
+completes. This can be done by using the Restore Backup GUI tool and selecting
+"Verify backup integrity, but do not restore the data." For details, see the 
+`QubesOS backup documentation <https://www.qubes-os.org/doc/backup-restore/>`_.
+
+Restore
+-------
+
+Reinstall QubesOS
+~~~~~~~~~~~~~~~~~
+
+To restore SecureDrop Workstation, follow our 
+:doc:`pre-install tasks <install>` to provision a QubesOS system complete with
+updated base templates.
+
+.. note::
+ If you backed up a number of custom VMs, during the installer, you may opt
+ to un-select "create default application qubes (personal, work, untrusted, vault)".
+
+Manage Unused VMs
+~~~~~~~~~~~~~~~~~
+
+If you installed all default application VMs, remove the ``vault`` VM, since we
+will replace it with the version in our backup.
+
+  .. code-block:: sh
+
+   qvm-prefs --set vault installed_by_rpm False 
+   qvm-remove vault
+
+Restore Backup
+~~~~~~~~~~~~~~
+
+Plug in your backup medium and unlock it as during the backup. By default
+on a new system, your peripheral devices will be managed by ``sys-usb``. 
+(For non-LUKS-based encryption media, you will need to install and configure the
+appropriate software, either in ``sys-usb`` or in another VM).
+
+Navigate to **Q ▸ System Tools ▸ Restore Backup**, and enter the  
+location of the backup file. You do not need to adjust the default Restore 
+options, unless you have made customizations to the backup. Enter the
+decryption/verification passphrase, and proceed to restore the backup.
+
+Your ``vault`` VM will be restored. 
+
+The contents of the previous ``dom0`` will be backed up to a folder in the 
+new ``dom0`` home directory called ``home-restore-<timestamp>/dom0-home/user``.
+In this guide we will refer to that location as ``$DOM0_BACKUP``.
+
+Open a terminal via **Q ▸ Terminal Emulator**: 
+
+  .. code-block:: sh
+
+   cd $DOM0_BACKUP/backup
+   cp -r securedrop-workstation-dom0-config/ /usr/share/
```

Comment:
:rotating_light: See note in backup section about the potentially harmful recursive copy here—we only want two (2) files. It's most important to specify them explicitly by fullpath in the backup section, but I'd still do the same in the restore section, to guard against surprises.

Reply:
thanks for this flag-- fixed in 05b8dfd